src/cmd/add.go

```diff
@@ -26,14 +26,20 @@ var addCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		if !template.HasTemplatesDir(appDir) {
-			fmt.Println("no templates found\n\nPlease add templates to the \".exosphere\" folder of your code base.")
-			os.Exit(1)
+		hasTemplates, err := template.HasTemplatesDir(appDir)
+		if err != nil {
+			panic(err)
```

Comment:
Yes, we should keep using panics for developer errors like  when an internal file isn't there. In the case here, unless I am misreading this, the error  is that there is no template directory in the repo. I consider that a user error, no?